### PR TITLE
i update the file laptopForm.js

### PR DESCRIPTION
**Pull request by Sslim123**
i update the input feld  placeholder , and pop-up alert when the submit the form 

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js [hunk 1]
```diff
@@ -51,15 +51,32 @@ function LaptopForm() {
...
+	const form = document.querySelector(".form");
+	//function display greeting after submit the from
+
 	function messageAlert() {
-		if (validateForm(document.laptopRequestForm)) {
-			alert(
-				"thank you for your completing the request form. your request have been recived "
-			);
-		}
+		console.log("heloo");
```
Comment (gregdyke) [inline client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js]: remove the console log before committing

File: client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js [hunk 2]
```diff
@@ -51,15 +51,32 @@ function LaptopForm() {
...
+		document.body.style.backgroundColor = "black";
```
Comment (gregdyke) [inline client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js]: I think this is OK, but I'm not sure. Does bootstrap not provide an API for this?

File: client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js [hunk 3]
```diff
@@ -51,15 +51,32 @@ function LaptopForm() {
...
+		modale.style.display = "block";
```
Comment (gregdyke) [inline client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js]: This is not OK: you may not modify the DOM that is rendered by the compo directly in React.

File: client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js [hunk 4]
```diff
@@ -51,15 +51,32 @@ function LaptopForm() {
...
+		form.style.display = "none";
+	}
+	function messageClosed() {
+		console.log("heloo");
+		modale.style.display = "none";
 	}
 	return (
 		<div className="form-card">
+			<div className="modale-content">
```
Comment (gregdyke) [inline client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js]: instead add the display block/none in here, based on a useState property that determines whether the form is visible or not

**Review by gregdyke — CHANGES_REQUESTED** [review]

File: client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js (hunk 1 above)
Comment (Sslim123) [reply client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js]: ok, i will do

File: client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js (hunk 2 above)
Comment (Sslim123) [reply client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js]: i don know, but it working

File: client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js (hunk 4 above)
Comment (Sslim123) [reply client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js]: do you mean i can make display block/none property with useState.

Comment (Sslim123) [reply client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js]: thank you for your feedback

File: client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js (hunk 3 above)
Comment (Sslim123) [reply client/src/pages/Home/Forms/LaptopRequest/LaptopForm.js]: i got you i will take it out,

**Review by gregdyke — APPROVED** [review]

**Comment by Sslim123** [comment]
thank for your help and approved my pull request

**Pull request merged**